### PR TITLE
Event Streams Working Draft 2

### PR DESCRIPTION
**Pull request by clemensv**
Addresses issues #6 and #5 

* Several clarifications
* Conformance clauses
* Security section

### CODE REVIEW

**Comment by k-wall** [comment]
@clemensv do you intend to commit the html?  Isn't source controlling the `.md` sufficient?

File: event-streams/src/event-streams-v1.0-wd01.md [hunk 1]
```diff
@@ -317,28 +324,31 @@ The model composes with AMQP link-level redirects as defined in [AMQP 1.0,
 attach a link, but rather point to a different AMQP container where the
 requested or assigned partition is available.
 
-If the event log assigns partition-bound links to consumers and wants to
-renegotiate those assignments, it MAY gracefully close some or all attached
-links and the respective producer or consumer SHOULD then attempt to establish a
-new link.
+Partition-bound links provide the consumer with the assurance that all
+transfers belong to the given partition, which is desireable when the
```
Comment (k-wall) [inline event-streams/src/event-streams-v1.0-wd01.md]: misspelt word
```suggestion
transfers belong to the given partition, which is desirable when the
```

File: event-streams/src/event-streams-v1.0-wd01.md [hunk 2]
```diff
@@ -436,24 +448,29 @@ If the consumer attaches a link scoped to a consumer group without specifying a
...
+2) The event log node MAY instead choose not to deny attaching the link
+   outright, but keep further incoming attach requests pending and complete
+   those only once a partition becomes again available for assignment. Attach
+   requests might therefore remain pending for a substantial time span, whose
+   limits MAY be configurable in an implementation specific way.
+
+If the event log node wants to force renegotiation of partition bindings amongst
+a consumer group, it SHOULD gracefully close all partition-bound links and the
+respective consumers SHOULD attempt to establish a new link, again with
+indicating the consumer group, but without specifying a partition.
```
Comment (k-wall) [inline event-streams/src/event-streams-v1.0-wd01.md]:
```suggestion
the consumer group, but without specifying a partition.
```

File: event-streams/src/event-streams-v1.0-wd01.md [hunk 3]
```diff
@@ -529,6 +546,18 @@ event log node before delivery to consumers.
...
 
+#### 4.6.4 event-streams-associated-partitions-changed delivery annotation
+
+The `event-streams-assigned-partitions-changed` delivery annotation is an array
+of `symbol` values and contains the list of partitions associated with the link
+that the transfer is being carried over.
+
+For all partition-agnostic consumer links, the event log node SHOULD add this
+annotation to the next transfer to the consumer once the association of
+partitions with the link has changed.
```
Comment (k-wall) [inline event-streams/src/event-streams-v1.0-wd01.md]: maybe?
```suggestion
annotation to the next transfer to the consumer every time the association of
partitions with the link changes.
```

**Comment by clemensv** [comment]
> @clemensv do you intend to commit the html? Isn't source controlling the `.md` sufficient?

I should a .gitignore then. Will do.

**Review by ppatierno — APPROVED** [review]

**Pull request merged**